include/interfaces/hydrogels_one_field.h

```diff
@@ -94,7 +94,7 @@ energies_and_residuals(const typename DoFHandler<dim,spacedim>::active_cell_iter
                        std::vector<std::vector<ResidualType> > &residuals,
                        bool compute_only_system_terms) const
 {
-  EnergyType alpha = this->alpha;
+  EnergyType alpha = 0;
```

Comment:
It is just a dummy number :) alpha is got inside the reinit function in order to "fix" y_dot

 

> Il giorno 05/mar/2016, alle ore 19:53, Luca Heltai <notifications@github.com> ha scritto:
> 
> In include/interfaces/hydrogels_one_field.h:
> 
> > @@ -94,7 +94,7 @@ energies_and_residuals(const typename DoFHandler<dim,spacedim>::active_cell_iter
> >                         std::vector<std::vector<ResidualType> > &residuals,
> >                         bool compute_only_system_terms) const
> >  {
> > -  EnergyType alpha = this->alpha;
> > +  EnergyType alpha = 0;
> Why do you go from alpha to 0, instead of this->get_alpha()?
> 
> —
> Reply to this email directly or view it on GitHub.
>